lib/glific/communications/message.ex

```diff
@@ -216,7 +217,7 @@ defmodule Glific.Communications.Message do
       sender_id: contact.id,
       receiver_id: Partners.organization_contact_id(organization_id),
       organization_id: contact.organization_id,
-      group_id: get_group_id(message_params)
+      group_id: create_or_get_group_id(message_params)
```

Comment:
call a function in separate line and store it in a variable and then use that variable here.. code looks cleaner